translate/src/hooks/useNarrowScreen.ts

```diff
@@ -1,21 +1,17 @@
 import { useEffect, useState } from 'react';
 
-const NARROW_SCREEN_MAX_WIDTH = 600;
-
 /**
- * Return true if the screen is narrower than 600px. Useful in Responsive Web Design.
+ * Return true if the screen is narrower than MAX_WIDTH. Useful in Responsive Web Design.
  */
-export function useNarrowScreen(): boolean {
-  const [isNarrow, setIsNarrow] = useState(
-    window.innerWidth <= NARROW_SCREEN_MAX_WIDTH,
-  );
+export function isNarrowerThan(MAX_WIDTH: number): boolean {
```

Comment:
Two problems here:
1. https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use
2. Exposing the `MAX_WIDTH` number means that our JS code gets littered with magic constants.

I'd suggest refactoring this as either
```js
useWindowWidth(): 'narrow' | 'mid' | 'wide'
```
or
```js
useViewportColumns(): 1 | 2 | 3
```